influxql/parser.go

```diff
@@ -311,8 +313,8 @@ func (p *Parser) parseCreateRetentionPolicyStatement() (*CreateRetentionPolicySt
 	// Parse optional DEFAULT token.
 	if tok, pos, lit = p.scanIgnoreWhitespace(); tok == DEFAULT {
 		stmt.Default = true
-	} else {
-		p.unscan()
+	} else if tok != EOF {
+		return nil, newParseError(tokstr(tok, lit), []string{"DEFAULT"}, pos)
```

Comment:
`FORCE` actually makes the code a lot easier IMO, as now you don't have to make any assumptions on the end user's behalf.

@dgnorton I'll add some tests for multiple statements and see what happens.